include/vctr/Containers/VctrBase.h

```diff
@@ -482,6 +482,38 @@ class VctrBase : private Config,
         return address >= first && address <= last;
     }
 
+    /** Returns the index of the first element that compares true to value or std::nullopt if none is found. */
+    template <std::equality_comparable_with<ElementType> T>
+    std::optional<size_t> indexOf (const T& value) const
+    {
+        auto it = find (value);
+        return it == end() ? std::nullopt : std::optional<size_t> (std::distance (begin(), it));
+    }
+
+    /** Returns the index of the last element that compares true to value or std::nullopt if none is found. */
+    template <std::equality_comparable_with<ElementType> T>
+    std::optional<size_t> indexOfReverse (const T& value) const
+    {
+        auto it = findReverse (value);
+        return it == rend() ? std::nullopt : std::optional<size_t> (std::distance (it, rend()) - 1);
```

Comment:
`rend()` is the reverse `end()`? Love it :D

Reply:
Yes, this is actually the standard library approach and naming convention for reverse iterators